test/remote/gateways/remote_ogone_test.rb

```diff
@@ -41,21 +41,21 @@ def test_successful_purchase_with_utf8_encoding_2
 
   # NOTE: You have to set the "Hash algorithm" to "SHA-1" in the "Technical information"->"Global security parameters"
   #       section of your account admin on https://secure.ogone.com/ncol/test/frame_ogone.asp before running this test
-  def test_successful_purchase_with_signature_encryptor_to_sha1
-    gateway = OgoneGateway.new(fixtures(:ogone).merge(:signature_encryptor => 'sha1'))
-    assert response = gateway.purchase(@amount, @credit_card, @options)
-    assert_success response
-    assert_equal OgoneGateway::SUCCESS_MESSAGE, response.message
-  end
+  # def test_successful_purchase_with_signature_encryptor_to_sha1
+  #   gateway = OgoneGateway.new(fixtures(:ogone).merge(:signature_encryptor => 'sha1'))
+  #   assert response = gateway.purchase(@amount, @credit_card, @options)
+  #   assert_success response
+  #   assert_equal OgoneGateway::SUCCESS_MESSAGE, response.message
+  # end
 
   # NOTE: You have to set the "Hash algorithm" to "SHA-256" in the "Technical information"->"Global security parameters"
   #       section of your account admin on https://secure.ogone.com/ncol/test/frame_ogone.asp before running this test
-  def test_successful_purchase_with_signature_encryptor_to_sha256
-    gateway = OgoneGateway.new(fixtures(:ogone).merge(:signature_encryptor => 'sha256'))
-    assert response = gateway.purchase(@amount, @credit_card, @options)
-    assert_success response
-    assert_equal OgoneGateway::SUCCESS_MESSAGE, response.message
-  end
+  # def test_successful_purchase_with_signature_encryptor_to_sha256
+  #   gateway = OgoneGateway.new(fixtures(:ogone).merge(:signature_encryptor => 'sha256'))
+  #   assert response = gateway.purchase(@amount, @credit_card, @options)
+  #   assert_success response
+  #   assert_equal OgoneGateway::SUCCESS_MESSAGE, response.message
+  # end
```

Comment:
Reading more I believe it's because you need to toggle a setting in the control panel to get either to pass...?